NEWS.md

```diff
@@ -1,5 +1,15 @@
-## 1.7.0 Unreleased
+## 1.8.0 Unreleased
 
+## 1.7.0 Released
+ Major changes in this release include applying the latest schema changes related to funds and acquisitions units.
+
+  [Full Changelog](https://github.com/folio-org/mod-gobi/compare/v1.6.0...v1.7.0)
+
+### Stories
+ * [MODORDSTOR-103](https://issues.folio.org/browse/MODORDSTOR-103) - Orders schema updates
+ * [MODORDERS-290](https://issues.folio.org/browse/MODORDERS-290) - Adding approve permission
```

Comment:
Please, move this story at top to follow ascending order.

Reply:
reordered stories